boards/iotlab-m3/include/periph_conf.h

```diff
@@ -62,8 +65,8 @@ extern "C" {
  */
 static const timer_conf_t timer_config[] = {
     /* device, APB bus, rcc_bit */
-    { TIM2, APB1, RCC_APB1ENR_TIM2EN, TIM2_IRQn },
-    { TIM3, APB1, RCC_APB1ENR_TIM3EN, TIM3_IRQn }
+    { TIM2, RCC_APB1ENR_TIM2EN, APB1, TIM2_IRQn },
+    { TIM3, RCC_APB1ENR_TIM3EN, APB1, TIM3_IRQn }
```

Comment:
I don't understand that change. And besides, why didn't you change it for the other boards?